### DISCUSSION TITLE
Performance issues, when adding components in root index

### DISCUSSION THREAD

**Discussion by aniketuplopwar** [vercel/next.js#60958] [Help]
### Summary

Hey All,

We are working with NextJs 13.5.6, with custom express(4.18.1) server and with page layout.

Our website development is currently in progress, and we developed quite a few pages and all look good in terms of performance.

Recently we added some component in our `src/pages/index.tsx` file, and out gatling tests against other pages (including index.tsx) started to show delays in response time. From average response time of 500ms it went up to 1100ms.

So basically our server side rendering of all the pages including the index page started to show delay.

We also tried removing the implementation and re-adding the components to confirm the problem, and it proved only when we add components in index route, it takes a hit on preformance

**NOTE:**  As said earlier these delays are only in the time taken on the server.

Has anyone encountered any such issue, or know the reason.

Thanks

**Comment by aniketuplopwar** [comment]
Found the root cause of it, which was outside the scope of next.

thanks
